wire-ios/Wire-iOS/Sources/AppRootRouter.swift

```diff
@@ -85,12 +84,6 @@ final class AppRootRouter: NSObject {
         sessionManager.switchingDelegate = switchingAccountRouter
         sessionManager.presentationDelegate = urlActionRouter
 
-        if !ProcessInfo.processInfo.isRunningTests {
-            if let countlyKey = Bundle.countlyAppKey, let host = BackendEnvironment.shared.countlyURL {
-                self.analyticsSession = AnalyticsSession(appKey: countlyKey, host: host)
```

Comment:
Why don't we check if we're running tests anymore?

Reply:
We don't init the analyticsSession In Wire-iOS anymore so there's no need for such piece of code there.

Reply:
But the analytics session can still be initialized in tests during tests, no?

Reply:
In our case they can't be since the analyticsConfiguration is going to be nil if that's nil there's no analytics session.

Reply:
Done: https://github.com/wireapp/wire-ios/commit/97f0562f9dbe96489fa05ee0355df9beb2204d65